publishing-service/src/main/avro/Product.avsc

```diff
@@ -6,7 +6,6 @@
     { "name": "gtin", "type": "string" },
     { "name": "name", "type": "string" },
     { "name": "alternateName", "type": [ "null", "string" ], "default": null },
-    { "name": "brand", "type": "string" },
```

Comment:
This deleted field is non-optional, but fortunately none of the consumers require it in their reader schema.